CHANGES.md

```diff
@@ -2,6 +2,10 @@
 
 ## [Unreleased]
 
+## [3.0.1] - 2024-11-14
```

Comment:
Note: I've created a `maint-3.0.x` branch because I think this can't wait before the next `minor` release, set to be done once we merge #155

Reply:
The changes will be back ported to the main branch later